cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -251,321 +279,6 @@ Slice::getTopLevelModule(const TypePtr& type)
     return getTopLevelModule(proxy ? dynamic_pointer_cast<Contained>(proxy) : dynamic_pointer_cast<Contained>(type));
 }
 
-void
```

Comment:
`trimLines`, `splitComment`, `parseCommentLine`, and `parseComment` here are identical to a set of functions already in the parser, except that they operate on a `DocElements` struct instead of a `Comment` class...

I removed all of this duplicate code, but had to switch everywhere that was using `DocElements` to use `CommentPtr` instead.